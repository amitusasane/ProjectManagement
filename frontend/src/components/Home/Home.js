import React from 'react';
import { Row, Col, Jumbotron, Container } from 'react-bootstrap';
import '../../scss/common.scss';

const Home = () => {
  return (
    <>
      <Container>
        <Row>
          <Col>
            <Jumbotron className="mt-5">
              <h1> Welcome to Project Management Tool!! </h1>
            </Jumbotron>
          </Col>
        </Row>
      </Container>
    </>
  );
};

export default Home;
