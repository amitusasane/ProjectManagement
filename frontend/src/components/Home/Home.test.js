import React from 'react';
import { MemoryRouter as Router } from 'react-router-dom';
import { render, cleanup } from '@testing-library/react';
import Home from './Home';

describe('Home Component', () => {
  afterEach(cleanup);
  it('Should render Home component', () => {
    const { container } = render(
      <Router>
        <Home />
      </Router>
    );
    expect(container).toMatchSnapshot();
  });
});
