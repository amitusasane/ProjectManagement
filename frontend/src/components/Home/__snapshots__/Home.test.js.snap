// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Component Should render Home component 1`] = `
<div>
  <div
    class="container"
  >
    <div
      class="row"
    >
      <div
        class="col"
      >
        <div
          class="mt-5 jumbotron"
        >
          <h1>
             Welcome to Project Management Tool!! 
          </h1>
        </div>
      </div>
    </div>
  </div>
</div>
`;
